app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def flash_message
+    flash.map do |key, message|
+      content_tag :p, message, :class => key
+    end.join().html_safe
```

Comment:
1. brackets for `join` are useless here
2. use `key: value` syntax
3. is it a `flash_message` or `flash_messages`?
4. I am not sure about this solution. Imho much more readable is to create flash view partial and put it into layout. Then you can iterate over a hash and put a proper paragraphs.